.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  pull_request:
+    branches: [ "main" ]
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  analysis:
+    name: Scorecard analysis
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+      # Needed to publish results and get a badge (see publish_results below).
+      id-token: write
+      # Uncomment the permissions below if installing in a private repository.
```

Comment:
Same here (like https://github.com/devfile/api/pull/1416/files/60c4de82d899148190ebfbef94811d26ccfb195e#r1445391059). I think we could drop all parts not related with our case :)

Reply:
I am in the process of cleaning up all the parts that aren't needed. Since in the future we are going to implement the fine-grained pat (or try to) so we can take advantage of the branch protection check, I was thinking of leaving that section commented in still, what do you think?

Reply:
Oh ok makes sense. I've also noticed https://github.com/devfile/api/issues/1417 :)

Last thing is about the comments for private repos inside the `scorecard.yaml`. I was thinking it might be better to remove them because they are not related with our case. Wdyt?

Reply:
I agree. I think I will remove all comments about private repos, leave the bit about PAT, and then leave the comments explaining why certain items are set which I think should be sufficient?

Reply:
Yep. I agree :)